addons/mail/models/ir_attachment.py

```diff
@@ -51,21 +55,22 @@ def _bus_notification_target(self):
         return self.env.user.partner_id
 
     def _attachment_format(self):
+        return [attachment._prepare_attachment_format(attachment) for attachment in self]
```

Comment:
Why the split? We lose the opportunity for multi records optimisation to be able to do an unnecessary override within the same module.